specification/billing/resource-manager/readme.md

```diff
@@ -29,6 +29,15 @@ openapi-type: arm
 tag: package-2020-05
 ```
 
+### Tag: package-2020-12-preview
+
+These settings apply only when `--tag=package-2020-12-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-12-preview'
+input-file:
+- Microsoft.Billing/preview/2020-12-15-preview/payNow.json
```

Comment:
as you removed billing.json, the APIs defined in that file will be missing in some SDKs. Is this behavior expected?

Reply:
Yes that's correct, as part of the 2020-12-15-preview we are considering the mentioned spec (payNow.json).

Reply:
Will all existing cmdlets in SDK will be broken?